webapp/WEB-INF/templates/admin/plugins/appointment/appointmentform/modify_appointmentform_tabs.html

```diff
@@ -317,6 +325,17 @@
 	}
 	else return true;
 	};
-	
+	function validateNbConsecutiveSlots() {
+		var nbConsecutiveSlotsInput = document.getElementById('nb_consecutive_slots');
+		var enteredValue = parseInt(nbConsecutiveSlotsInput.value);
+
+		if (enteredValue < 2 || isNaN(enteredValue)) {
```

Comment:
Why the restriction of multi-slot calendars to slots with a number of places greater than 2 ?

Reply:
It is possible to implement a multi-slot form that includes one or more consecutive slots.
The imposed limitation of having at least 2 consecutive slots in multi-slot forms is not desired. This restriction will result in unnecessary regressions.